src/components/search/instructors/instructor-essential.tsx

```diff
@@ -48,7 +48,7 @@ const SearchInstructorEssential: FunctionComponent<InstructorProps> = ({
       />
       <div className="items-center flex flex-col grid-cols-1 space-y-12 lg:grid lg:grid-cols-12 lg:space-y-0 dark:bg-gray-900">
         <div
-          className={`w-full lg:col-span-8 bg-white dark:bg-gray-800 dark:text-gray-200 shadow-sm h-full relative items-start overflow-hidden grid lg:grid-cols-8 lg:gap-2 ${
+          className={`w-full lg:col-span-8 dark:text-gray-200 h-full relative items-start overflow-hidden grid lg:grid-cols-8 lg:gap-2 ${
```

Comment:
removed backgound/shadow so that the banner misalignment go away visually:

![Image 2022-04-21 at 1 30 56 PM](https://user-images.githubusercontent.com/1519448/164439713-ded89563-ffa2-458e-b47c-5f5c08a94c84.jpg)